### ISSUE TITLE
Add import from 'g' files

### ISSUE THREAD

**Issue by jkennedy-usgs** [opengrav/gmeterpy#12]
It would be useful to import processed g values output from 'g' software. If nothing else, this would be useful as a placeholder to develop other functionality until we can calculate g from fringe intervals.

There's at least two options: import the pseudo-xml in the .fg5.txt file generated from the Export command on the 'g' file meanu, or import from the .project.txt file. The former has the advantage of being easy to parse and having some information not in the .project.txt file. The latter has the advantage of being generated each time a file is run, without the extra "Export" step. 

Do we expect the overall design of gmeterpy to be object-oriented, in the sense there will be an "absolute_gravity" object? This object could be created from a 'g' file, or from raw observations, etc. I have [code ](https://github.com/jkennedy-usgs/sgp-utils/blob/master/sgp-utils/fg5.py)that can generate this type of object from a .project.txt file.

**Comment by ioshchepkov** [comment]
@jkennedy-usgs yes, in general, the design is expected to be object-oriented, but first we will need carefully plan how it should be realized. Especially for the instruments part. We need a more general view on how to wok with different gravimeters. In my opinion the best thing we can do is to implement plugin system. But first it is needed to write basic data structure and it will take some time.

Thank you for your suggestion about 'g' data parser. I am planning to realize it, but it needs to be consistent also with the raw data parser and with the basic data structure, at least txt raw data (from the Export function). Basically, your Issue should be more general and mention the fringes and I'm OK with that.

**Comment by jkennedy-usgs** [comment]
Okay, sounds good. I'll wait until there's a little more structure before contributing.
